### DISCUSSION TITLE
How to use H5P Nodejs with custom application?

### DISCUSSION THREAD

**Discussion by himanshuahuja96** [Lumieducation/H5P-Nodejs-library#2471] [Q&A] [answered]
Hello there,

Thank you so much for such great work. We are building a learning management system for our company and would like to have H5P integrated into the system. Can you please help us where to look and see how we can integrate it into our project?

**Comment by sr258** [comment, marked as answer]
@himanshuahuja96 The online documentation is the best staring place: https://docs.lumi.education/

**Comment by thuanjdev** [comment]
Hello. I want to change the language to Vietnamese using nodejs (https://github.com/Lumieducation/H5P-Nodejs-library). Who can help me?

**Reply by sr258** [reply]
Have you read [the documentation on this](https://docs.lumi.education/advanced-usage/localization)? It might be possible that H5P isn't fully available in Vietnamese, though. You'll have to check. There is a translation platform at https://translate-h5p.tk/

**Reply by thuanjdev** [reply]
Thanks for your support! I found it on https://translate-h5p.tk/weblate/projects/. How to embed it in my nodejs project!

**Reply by sr258** [reply]
Please read the documentation I've mentioned. It explains how to enable localization.

**Comment by Mukul96-op** [comment]
@himanshuahuja96  are you able to integrate h5p on your app?
